string name
geometry_msgs/Pose2D[] poses
